tests/aws/services/lambda_/test_lambda.py

```diff
@@ -1046,6 +1121,23 @@ def test_lambda_url_invocation_exception(self, create_lambda_function, snapshot,
         assert to_str(result.content) == "Internal Server Error"
         assert result.status_code == 502
 
+    @markers.aws.validated
+    def test_lambda_url_invalid_invoke_mode(self, create_lambda_function, snapshot, aws_client):
+        function_name = f"test-fn-echo-{short_uid()}"
+
+        create_lambda_function(
+            func_name=function_name,
+            zip_file=testutil.create_zip_file(TEST_LAMBDA_PYTHON_ECHO_JSON_BODY, get_content=True),
```

Comment:
`handler_file=TEST_LAMBDA_PYTHON_ECHO_JSON_BODY,` automates `zip_file` and `handler`